django/contrib/gis/geoip2.py

```diff
@@ -25,19 +25,12 @@
 
 try:
     import geoip2.database
-except ImportError:
+except ImportError:  # pragma: no cover
```

Comment:
Do we need this? :thinking:

Reply:
I was improving test coverage for this module and this is just something that isn't easily testable.

This stops the line below from showing as uncovered.